packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -118,6 +118,8 @@ export const makeOfferSpecShape = (bidBrand, collateralBrand) => {
 export const prepareAuctionBook = (baggage, zcf, makeRecorderKit) => {
   const makeScaledBidBook = prepareScaledBidBook(baggage);
   const makePriceBook = preparePriceBook(baggage);
+  // Brands that have or are making active quoteNotifier Observers
+  const observedBrands = new Set();
```

Comment:
I'm struggling to get the relevant invariant in my head.

Something that combines `observedBrands` with `state.updatingOracleQuote`